### DISCUSSION TITLE
Adding CodeExchange icon

### DISCUSSION THREAD

**Discussion by rosemaryluo** [twilio-labs/paste#2472] [Request: Icon] [answered]
Hi, what is the process for getting an icon added to https://paste.twilio.design/components/icons/ ? I tried to go to https://paste.twilio.design/components/icons/how-to-add-an-icon as instructed but I don't think that link works anymore.

This is the CodeExchange logo we are trying to add:
https://library.twilio.com/media/?mediaId=213A61F1-E910-44AC-A3D9315B5B0B172D

**Comment by SiTaggart** [comment]
Hi @rosemaryluo 

The correct link is https://paste.twilio.design/introduction/contributing/icons 

Curious if you can point us to where you found that non-functional link, it'd be great to get it updated.

**Reply by rosemaryluo** [reply]
Saw it when I tried to select a category for the github discussion :) 

<img width="454" alt="image" src="https://user-images.githubusercontent.com/87049408/173958130-d5432dab-c6f2-4c15-b721-0d82613e254c.png">

**Reply by rosemaryluo** [reply]
This is a logo for CodeExchange which we are using only on CodeExchange at the moment, we don't have any current integrations into Console where other teams would need access (that I know of).

**Comment by loreina** [comment, marked as answer]
Hi @rosemaryluo,

Sorry about the link issue, the category description's been updated with the correct one. Appreciate you catching that and helping us to improve our resources. 

For contributing the icon, adding the CodeExchange logo as a product icon in Paste makes sense to me. Feel free to follow the guidelines on that page and share a link to your Figma branch in this discussion. From there, a couple designers on our team will review the branch and help to publish the icon in the design library. Let me know if you have any questions!

**Reply by rosemaryluo** [reply]
Of course!

Hope this is right:
https://www.figma.com/file/E6KUvMhioUmAgN0nwmReTM/branch/mCfXyEY51c5HNDPEkvgfOR/Paste-Components?node-id=11832%3A56760

**Reply by serifluous** [reply]
Thanks, Rosemary! The branch is currently in review. When it's approved, are you planning on having your team [submit a PR](https://paste.twilio.design/introduction/contributing/icons/#adding-the-icon-to-the-paste-repository) (can be more timely for your engineering team), or have us do it (2-6 week wait)?

**Reply by rosemaryluo** [reply]
will check and get back to you!

**Reply by rosemaryluo** [reply]
https://issues.corp.twilio.com/browse/GROW-3809

**Reply by serifluous** [reply]
Thanks! The Figma branch has been approved and merged so it's available in the Paste Components library now.
